classes/quick-editor/class-quick-editor.php

```diff
@@ -49,17 +49,20 @@ public function end_capture_page() {
 			return;
 		}
 
+		$personal_options = __( 'Personal Options' );
+
 		// Move user information to top
-		preg_match( '@<h2>(.*?)</table>@s', $profile_page, $profile_details );
+		preg_match( '@<h2>' . $personal_options . '</h2>(.*?)</table>@s', $profile_page, $profile_details );
```

Comment:
Do you think it is worth doing a `preg_quote( $personal_options, '@' )` here? I'm not sure if any translations would have a regex-y character (probably not?).

Reply:
It can't hurt, added.